ast/term.go

```diff
@@ -229,7 +239,15 @@ func ValueToInterface(v Value, resolver Resolver) (interface{}, error) {
 // JSON returns the JSON representation of v. The value must not contain any
 // refs or terms that require evaluation (e.g., vars, comprehensions, etc.)
 func JSON(v Value) (interface{}, error) {
-	return ValueToInterface(v, illegalResolver{})
+	return JSONOrdered(v, false)
+}
+
+// JSONOrdered returns the JSON representation of v. The value must not contain any
+// refs or terms that require evaluation (e.g., vars, comprehensions, etc.) If sortSets is true
+// this function will sort sets before converting them to JSON arrays (which makes it more
+// expensive.)
+func JSONOrdered(v Value, sortSets bool) (interface{}, error) {
```

Comment:
If we introduce another function, we could have an opts vararg, in case we'd like to add more later? Named `JSONwithOpts` or something... 💭